aws/src/main/java/org/apache/iceberg/aws/AwsCatalogProperties.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aws;
+
+public class AwsCatalogProperties {
+
+  private AwsCatalogProperties() {
+  }
+
+  /**
+   * Type of S3 Server side encryption used, default to {@link AwsCatalogProperties#S3_SSE_TYPE_NONE}.
+   */
+  public static final String S3_SSE_TYPE = "s3-sse-type";
+
+  /**
+   * No server side encryption.
+   */
+  public static final String S3_SSE_TYPE_NONE = "none";
```

Comment:
These four types might make sense to turn into an enumeration.  Also, are these official names for the encryption types?  I just find `s3` not to be terribly descriptive for the type of encryption.

Reply:
Yeah, but unfortunately that's the official name. I am hesitated to turn them into enums, since it seems like in Iceberg people just use string instead of enum in many different places.